src/Algolia.Search/Algolia.Search.csproj

```diff
@@ -20,16 +21,11 @@
     <Copyright>Copyright 2019 Algolia</Copyright>
     <Version>6.7.0</Version>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard2.1;netstandard2.0;netstandard1.3</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netstandard2.1;netstandard2.0;netstandard1.3;net45;</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;netstandard1.3</TargetFrameworks>
     <LangVersion>7.3</LangVersion>
   </PropertyGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3' OR '$(TargetFramework)' == 'netstandard2.0' OR '$(TargetFramework)' == 'netstandard2.1'">
+  <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="12.0.2"/>
-  </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'net45'">
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.2"/>
-    <Reference Include="System"/>
-    <Reference Include="System.Net.Http"/>
+    <None Include="images\icon.png" Pack="true" PackagePath="\"/>
```

Comment:
What is it used for? The AlgoliaSearch package's icon?

Reply:
Ok I think I got after reading your PR's description, it was the warning wrote by `dotnet pack`.

Reply:
Described in the commit message: https://github.com/algolia/algoliasearch-client-csharp/pull/717/commits/4785037114fd345519a681257890edd3315465c0.

It instructs `dotnet pack` where to find the `icon.png` file locally (because the URL fetching method is now deprecated). That's why I've added the PNG file in the commit as well.